src/jolt/gui/jolt_app.py

```diff
@@ -143,7 +143,8 @@ def __init__(self, simulated=TEST_NOHW):
     def load_config(self):
         """
         Reads configuration file
-        :returns: (float, float, float, str): voltage, gain, offset, channel
+        :returns: (float, float, float, str, bool): voltage, gain, offset, channel, ambient (False means more
+        restrictions on the permitted values, e.g. for the pressure)
```

Comment:
Could you explain more what ambient=True means? I'd actually suggest to explain it in the case of True, as False is the default and normal behaviour. You've explained it in the commit log, but it should be also here, for clarity.